src/driver/postgres/PostgresDriver.ts

```diff
@@ -736,7 +736,15 @@ export class PostgresDriver implements Driver {
                 || tableColumn.srid !== columnMetadata.srid;
         });
     }
-
+    private lowerDefaultValueIfNessesary(value: string | undefined) {
+        // Postgres saves function calls in default value as lowercase #2733
+        if (!value) {
+            return value;
+        }
+        return value.split(`'`).map((v, i) => {
```

Comment:
just want a confirmation - are you sure it is **safe**?

Reply:
I haven't found anything in the docs if function calls are always lowercase - but it definitely looks that way. I've found that one scenario which could be broken this is: if someone use posix escape string syntax, but it isn't db default behavior and that's generally not preferable. https://stackoverflow.com/questions/12316953/insert-text-with-single-quotes-in-postgresql